### ISSUE TITLE
Oracle ODP.NET Logging Module

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#181]
```
I am proposing that a separate class be added to log errors in an Oracle 
database through Oracle's ODP.NET library.

I have modified the OracleErrorLog class to use ODP.NET instead of the now 
deprecated System.Data.OracleClient. I am attaching the code to this post.
```

Original issue reported on code.google.com by `JRusba...@gmail.com` on 24 Aug 2010 at 7:14

Attachments:
- [OracleErrorLog.cs](https://storage.googleapis.com/google-code-attachments/elmah/issue-181/comment-0/OracleErrorLog.cs)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for this!
```

Original comment by `jamesdriscoll71` on 25 Aug 2010 at 9:06
- Changed state: **Accepted**
- Added labels: **Component-Persistence**, **DotNet-4.0**, **Milestone-Release2.0**

**Comment by GoogleCodeExporter** [comment]
```
I should specify that I am only able to test my changes against an Oracle 10g 
database, using ODP.NET version 10.2.0.2 (assembly version 2.102.2.20).
```

Original comment by `JRusba...@gmail.com` on 25 Aug 2010 at 3:02

**Comment by GoogleCodeExporter** [comment]
```
I have very limited experience with ODP.NET but I am nonetheless familiar with 
version mismatch headaches[1]. Has there been any though about how would a 
library like ELMAH go about being as independent of the Oracle client 
deployment?

[1] 
http://stackoverflow.com/questions/3692153/oracle-odp-net-version-agnostic-alter
native
```

Original comment by `azizatif` on 25 Dec 2011 at 1:04

**Comment by GoogleCodeExporter** [comment]
```
Started in http://code.google.com/r/jamesdriscoll71-odpnet/
```

Original comment by `jamesdriscoll71` on 17 Mar 2012 at 2:18
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b5de13ec6ef4.
```

Original comment by `azizatif` on 19 Jun 2012 at 7:58
- Changed state: **Fixed**
